common-app-lib/src/cmr/common_app/api/routes.clj

```diff
@@ -134,7 +137,9 @@
             ;; _ (println "GETTING CACHE " (keyword cache-name))
             cache (cache/context->cache request-context (keyword cache-name))
             ;; _ (when (nil? cache) (println "CACHE IS NIL"))
-            result (cache/get-value cache cache-key)]
+            result (case (str (type cache))
```

Comment:
No, I wasn't able to, I think it is because in a case you can't do any processing when comparing. Using cond you can.  I changed this anyway.  First I used your suggestion of using instance?  and second I am now using an if statement instead since there are only 2 options anyway.